Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -18,13 +22,15 @@
 SEVERITY_OPTIONS = {
     'Low': 0,
     'Medium': 1,
-    'High': 2
+    'High': 2,
+    'All': [0, 1, 2]
 }
 
 RESOLUTION_STATUS_OPTIONS = {
```

Comment:
In the daily you said something about supporting multiple options, but here you only support all

Reply:
I added the function `params_to_filter` that handle it